devel/wolfssl/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                wolfssl
-version             5.4.0
-revision            1
+version             5.5.0
+revision            2
```

Comment:
When `version` increments, `revision` needs to reset to 0.

Reply:
Thanks @herbygillot updating the commit now